cli/src/lib/zk-sync-era-state.ts

```diff
@@ -120,22 +129,15 @@ export class ZkSyncEraState {
     return diff;
   }
 
-  async getCurrentSystemContractData(addr: Hex): Promise<SystemContractData> {
-    const client = createPublicClient({
-      transport: http("https://mainnet.era.zksync.io"),
-    });
+  async getCurrentSystemContractBytecodeHash(addr: Hex): Promise<string> {
```

Comment:
This method now is simpler. It just returns the hash. And if the contract was not found it returns "Not found". I don't love this logic, but it works.